### PR TITLE
drivers/modem/alt1250: Fix poll function

### PR DESCRIPTION
**Pull request by SPRESENSE**
## Summary
Notify only when there are available events.

## Impact
modem/alt1250 only

## Testing
spresense lte works well.

### CODE REVIEW

**Review by xiaoxiang781216 — APPROVED** [review]

File: drivers/modem/alt1250/alt1250.c [hunk 1]
```diff
@@ -279,6 +279,25 @@ static void write_evtbitmapwithlist(FAR struct alt1250_dev_s *dev,
   nxmutex_unlock(&dev->evtmaplock);
 }
 
+/****************************************************************************
+ * Name: is_evtbitmap_avail
+ ****************************************************************************/
+
+static int is_evtbitmap_avail(FAR struct alt1250_dev_s *dev)
```
Comment (pkarashchenko) [inline drivers/modem/alt1250/alt1250.c:286]:
```suggestion
static bool is_evtbitmap_avail(FAR struct alt1250_dev_s *dev)
```

File: drivers/modem/alt1250/alt1250.c [hunk 2]
```diff
@@ -279,6 +279,25 @@ static void write_evtbitmapwithlist(FAR struct alt1250_dev_s *dev,
...
+{
+  int ret;
```
Comment (pkarashchenko) [inline drivers/modem/alt1250/alt1250.c:288]:
```suggestion
  bool ret;
```

File: drivers/modem/alt1250/alt1250.c (hunk 1 above)
Comment (SPRESENSE) [reply drivers/modem/alt1250/alt1250.c:286]: Fix with https://github.com/apache/incubator-nuttx/pull/7408

File: drivers/modem/alt1250/alt1250.c (hunk 2 above)
Comment (SPRESENSE) [reply drivers/modem/alt1250/alt1250.c:288]: Fix with https://github.com/apache/incubator-nuttx/pull/7408

**Pull request merged**